### ISSUE TITLE
New JBrowse / GBrowse track

### ISSUE THREAD

**Issue by gw3** [WormBase/website#4789]
Could we have a new track added to the genome browsers?
The data is for Trinity sequences aligned to the genome using BLAT.
Trinity sequences are contigs of selected RNASeq reads constructed using the program "Trinity".

The Trinity data in the GFF files has the Source name 'BLAT_Trinity_BEST' and the Feature name 'expressed_sequence_match' .

The track should be displayed in the section "Transcription", sub-section "Supporting Evidence", together with the EST and other similar tracks.

Thanks,
Gary

**Comment by scottcain** [comment]
Sure, I'll add those today, though with the changes made to categories recently, this would go under Genes -> Supporting Evidence.

**Comment by scottcain** [comment]
@gw3 Here's my first pass at the track in JBrowse:

http://staging.wormbase.org/tools/genome/jbrowse-simple/index.html?data=data%2Fc_elegans_PRJNA13758&loc=IV%3A6362216..6375836&tracks=Trinity-assembled%20RNAseq%2CmRNAs_ncRNAs%20(best)%2CESTs%20(best)%2CC.elegans%20RSTs%2CC.%20elegans%20OSTs&highlight=

A few items I'd like feedback on:
* Do you have a preferred color (I used the purple that is close to the purple in the Trinity logo)?
* Do you have somewhere that links can go to for individual features?
* I used this text for the "About this track": "Native (same species) RNAseq data was assembled with Trinity and then aligned to the genome with BLAT"  Is that acceptable or would you like to provide more information?
* Would you like the labels trimmed in some way (like trimming "elegans" and other characters off the beginning)?
* What species currently have these features (WS253)?

Thanks

**Comment by gw3** [comment]
@scottcain : 
 - the colour is good
 - there are no links required
 - The description is good.
 - the labels should not be trimmed: this is their ID
 - elegans, brugia, briggsae, ovolvulus, the other species will be added when we include them in a database release Build. 

Thanks

**Comment by scottcain** [comment]
Hi @gw3 I added the tracks to GBrowse as well, though I only found those data types in the elegans and volvulus GFF3 files--perhaps the other species didn't make the cut for this release.  I'm generally pretty happy with how it turned out, though I note that oddly, GBrowse and JBrowse seem to have very different ideas of what "mediumpurple" should look like (it's a not-unattrative blue in GBrowse), but if that doesn't bother you, I'll leave it as is.  Otherwise, feel free to close this issue.  Cheers!
